src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md

```diff
@@ -5,46 +5,76 @@ head_title: 'xk6-disruptor Documentation'
 excerpt: 'xk6-disruptor is a k6 extension providing fault injection capabilities to test system reliability under turbulent conditions.'
 ---
 
-xk6-disruptor is a k6 extension that can inject faults into a system to simulate turbulent conditions. 
+[xk6-disruptor](https://github.com/grafana/xk6-disruptor) is an extension that adds fault injection capabilities to k6. It implements the principles of the Chaos Engineering discipline to test the reliability of our applications under turbulent conditions such as delays and response errors.
 
-xk6-disruptor intends to help testers approach the discipline of "Chaos Engineering" with k6 way&mdash;providing the best developer experience as a primary objective. 
+Key features include:
```

Comment:
This list is rendered with substantial spacing between bullet points, while the one below is tightly cuddled. We should probably settle for one style (cudded looks better to me).

Reply:
fixed by https://github.com/grafana/k6-docs/pull/1233/commits/3f43a2aff34bd687ff3083956bf324874989820a